backend/wmg/data/wmg_cube.py

```diff
@@ -175,6 +177,7 @@ def build_in_mem_cube(gene_ids, cube_index, other_attrs, cube_sum, cube_nnz):
         dims[2][idx : idx + n_vals] = organism_ontology_term_id
 
         vals["sum"][idx : idx + n_vals] = cube_sum[cube_idx, mask]
+        vals["sumsq"][idx : idx + n_vals] = cube_sq_sum[cube_idx, mask]
```

Comment:
Does this actually get added to the cube? I figured we'd need to add a TileDB schema attribute [here](https://github.com/chanzuckerberg/single-cell-data-portal/blob/1dd0fac3c57c7d2c385171e75282a6c442824697/backend/wmg/data/schemas/cube_schema.py#L40).  Even if this somehow working without that attribute defined, I would add there. 

In the schema, I would comment how/why this attribute is being used (it's not for WMG iirc?).

A unit test should be added/updated to assert that the attribute is being stored.